pkg/ibmvpc/vpc.go

```diff
@@ -378,22 +443,24 @@ func (n *NACL) AllowedConnectivity(src, dst vpcmodel.Node, isIngress bool) (*com
 	return n.analyzer.AllowedConnectivity(subnetCidr, inSubnetCidr, targetNode.Cidr(), isIngress)
 }
 
-func (n *NACL) RulesInConnectivity(src, dst vpcmodel.Node, conn *common.ConnectionSet,
-	isIngress bool) (allow, deny []int, err error) {
-	targetNode, subnetCidr, inSubnetCidr, err := n.initConnectivityComputation(src, dst, isIngress)
-	if err != nil {
-		return nil, nil, err
+func (n *NACL) rulesInConnectivity(src, dst vpcmodel.Node, conn *common.ConnectionSet,
+	isIngress bool) (tableRelevant bool, allow, deny []int, err error) {
```

Comment:
in some places we have rulesInConnectivity() and in other with capital R RulesInConnectivity()
is it on purpose? 
if yes, I think it is a bad practice

Reply:
Yes; in the layer level it should be imported.
Renamed to avoid this lower case and upper case dupication